components/panel/body.js

```diff
@@ -52,7 +52,10 @@ class PanelBody extends Component {
 							aria-expanded={ isOpened }
 						>
 							<Dashicon icon={ icon } />
-							{ title }
+							<span className="components-panel__body-title__text-span">
```

Comment:
Would recommend to avoid including an element name in a class name, since it's incidental and can change over time, isn't representative of the semantic meaning of the element.